cylc/flow/etc/global.cylc.eg

```diff
@@ -1,23 +0,0 @@
-#------------------------------------------------------------------------------
-# How to create a site or user global.cylc global config file.
-#------------------------------------------------------------------------------
-# The "cylc config" command prints global config defaults,
-# overridden by site global settings (if any), overridden by user global
-# settings (if any).
-#
-# To generate a new global config file:
-#   % cylc config > global.cylc
```

Comment:
File not referenced from the docs so no one is likely to find it.

The advice to generate a global config from the hardcoded defaults is something we probably shouldn't push ahead with.